v2/tools/generator/internal/codegen/pipeline/create_resource_extension_types.go

```diff
@@ -53,10 +57,5 @@ func CreateResourceExtensions(localPath string, idFactory astmodel.IdentifierFac
 			return state, nil
 		})
 
-	// We don't want tests to be generated for resourceExtensions, since these are not the actual resource types.
-	// Therefore, we want to make sure that 'createResourceExtensions' stage only runs when these prerequisite
-	// stages have completed.
-	stage.RequiresPrerequisiteStages(InjectJsonSerializationTestsID, InjectPropertyAssignmentTestsID)
```

Comment:
Don't we need this anymore?

Reply:
We do need the run-order dependency - I moved it to the other end, using `RequiresPostrequisiteStages()`; this allows us to test this stage _without_ having the test injection stages present.